README.md

```diff
@@ -44,7 +44,7 @@ Example command for not yet released version of Red Hat build of Quarkus:
 
 ```
 mvn -fae clean verify \
- -Dts.use-ephemeral-namespaces -Dts.authenticated-registry \
+ -Dts.authenticated-registry \
  -Dversion.quarkus=1.7.1.Final-redhat-00001 \
  -Dquarkus.platform.group-id=com.redhat.quarkus \
  -Dmaven.repo.local=/Users/rsvoboda/Downloads/rh-quarkus-1.7.1.GA-maven-repository/maven-repository
```

Comment:
Add `#running-tests-in-ephemeral-namespaces` pointer (below this command example) to `Running tests in ephemeral namespaces` section

Reply:
Added. Let me know if it looks better now.